target_actionnetwork/target.py

```diff
@@ -1,26 +1,23 @@
 """ActionNetwork target class."""
 
-from __future__ import annotations
-
-from singer_sdk import typing as th
 from singer_sdk.target_base import Target
 from target_hotglue.target import TargetHotglue
 
 from target_actionnetwork.sinks import (
     ContactsSink,
 )
 
-SINK_TYPES = [
-    ContactsSink,
-]
-
 class TargetActionNetwork(Target, TargetHotglue):
     """Sample target for ActionNetwork."""
 
     name = "target-actionnetwork"
 
     MAX_PARALLELISM = 1
 
+    SINK_TYPES = [
```

Comment:
Any particular reason why you made this change?

Reply:
no, put it back